Editor/UI/TreeViews/TreeViewSelection.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.Editor
+{
+    public class TreeViewSelection
+    {
+        public List<string> groups;
+        public List<string> selection;
+
+        public TreeViewSelection()
+        {
+            groups = new List<string>();
+            selection = new List<string>();
+        }
+
+        public TreeViewSelection(TreeViewSelection threadSelection)

```

Comment:
threadSelection needs to be renamed (to treeViewSelection?)

Reply:
Done. I'll push this to the head of this branch and try to figure out how to update this pull request with the changes.